pkgs/by-name/zu/zulip-emoji-mapping/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "zulip-emoji-mapping";
+  version = "1.0.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "GearKite";
+    repo = "zulip-emoji-mapping";
+    rev = "v${version}";
+    hash = "sha256-logm5uAnLAcFqI7mUxKEO9ZmHqRkd6CFiCW4B5tqZzg=";
+  };
+
+  buildInputs = with python3Packages; [
+    setuptools
```

Comment:
Just ask for clarification, this isn't working in `build-system` or `dependencies`?

Reply:
I'm not sure what you mean. Do you mean that we could also use `dependencies = [ ... ]`?

Reply:
Is it okay to remove?

Reply:
I see. 
No, I just tried it and it failed building. So it seems it is needed.

Reply:
I don't think it's called in runtime. Maybe it should be in `build-system`. What do you think of it?

Reply:
You are right, that also works perfectly fine! I added it.